P5/Source/Guidelines/en/ND-NamesDates.xml

```diff
@@ -1035,8 +1034,7 @@ target="#CONA"/>).  The tag -->The
           <!-- socecstatus has its own @code and @scheme attributes which need
 exemplifying here too? -->
         </p>
-        <p>The meanings of concepts such as sex, nationality, or age are highly culturally-dependent, and the encoder should take
-          particular care to be explicit about any assumptions underlying their usage of them. For example, when recording
+        <p>Encoders should take particular care to be explicit about culturally determined concepts such as sex/gender system, nationality, ethnicity, or age that influence their markup. For example, when recording
```

Comment:
What is a “sex/gender system”? Is it a culturally determined concept? Either way I hate using a slash like that. I think I would prefer to just see “sex, gender, nationality, ethnicity, or age”.

Reply:
I believe this is in reference to Gayle Rubin's influential theory of "sex/gender systems". Gayle Rubin, "The Traffic in Women: Notes on the ‘Political Economy’ of Sex,” Toward an Anthropology of Women, ed. Rayna R. Reiter (New York: Monthly Review Press, 1975) 157-210. 
Perhaps there should be a reference here so that it's clear that we are borrowing a term

Reply:
I've modified the wording a little and added a note with reference to Gayle Rubin.